__tests__/Page_test.re

```diff
@@ -0,0 +1,15 @@
+open Jest;
+open Expect;
+open Game;
+
+describe("re-dune", () =>
```

Comment:
The file name should be the same as file tested (`Game_test.re`).

The output of example bellow will be: "module Game method createPlayingBoard will create 2x3 playing board". It's better than output: "re-dune create playing board of 2".
```javascript
describe("module Game", () => {
  describe("method createPlayingBoard", () => {
    test("will create 2x3 playing board", () => { ... })
  })
})

Reply:
thanks, I'm getting use to writing tests :)